product_docs/docs/net_connector/8.0.5.1/02_requirements_overview.mdx

```diff
@@ -18,3 +16,22 @@ The EDB .NET Connector graphical installers are supported on the following Windo
 32-bit Windows:
 
 -   Windows 10
+
+## Suported database server versions
+
+This table lists the latest EDB .NET Connector versions and their supported corresponding EDB Postgres Advanced Server (EPAS) versions. 
+
+| EDB .NET Connector                           | EPAS17 | EPAS16 | EPAS 15 | EPAS 14 | EPAS 13 | EPAS 12 |
+|----------------------------------------------|--------|--------|---------|---------|---------|---------|
+| [8.0.5.1](release_notes/8.0.5.1_rel_notes)   | Y      | Y      | Y       | Y       | Y       | Y       |
+| [8.0.2.1](release_notes/8.0.2.1_rel_notes)   | N      | Y      | Y       | Y       | Y       | Y       |
+| [7.0.6.2](release_notes/7.0.6.2_rel_notes)   | N      | Y      | Y       | Y       | Y       | Y       |
+| [7.0.6.1](release_notes/7.0.6.1_rel_notes)   | N      | Y      | Y       | Y       | Y       | Y       |
+| [7.0.4.1](release_notes/7.0.4.1_rel_notes)   | N      | N      | Y       | Y       | Y       | Y       |
+| [6.0.2.1](release_notes/6.0.2.1_rel_notes)   | N      | N      | N       | Y       | Y       | Y       |
+| [5.0.7.1](release_notes/5.0.7.1_rel_notes)   | N      | N      | N       | N       | Y       | Y       |
+| [4.1.6.1](release_notes/4.1.6.1_rel_notes)   | N      | N      | N       | N       | Y       | Y       |
+| [4.1.5.1](release_notes/4.1.5.1_rel_notes)   | N      | N      | N       | N       | Y       | Y       |
+| [4.1.3.1](release_notes/4.1.3.1_rel_notes)   | N      | N      | N       | N       | Y       | Y       |
+| [4.0.10.1](release_notes/4.0.10.1_rel_notes) | N      | N      | N       | N       | N       | Y       |
```

Comment:
In Confluence we have 4.0.10.2 and 4.0.10.1 instead of 4.0.10.1 and 4.0.6.1 like here. How should we proceed?

Reply:
4.0.6.1 was missing in the Confluence page, thanks